src/lib/email-service.js

```diff
@@ -2,58 +2,47 @@
 const nodemailer = require("nodemailer")
 const config = require('../config')
 
-module.exports = class Email {
-	/** Represents the email sender
-     * @constructor
-     * @param {object} user - The user object 
-     */
-	constructor(user) {
-		this.to = user.email
-		this.firstName = user.firstName
-		this.from = config("EMAIL_FROM")
-	}
 
-	/**
-     * Selectively creates a new transport based on the environment
-     * @returns a new nodemailer transport
-     */
-	newTransport() {
-
-		if (config('NODE_ENV') === 'development') {
-			return nodemailer.createTransport({
-				host: config('EMAIL_HOST'),
-				port: config('EMAIL_PORT'),
-				auth: {
-					user: config('EMAIL_USERNAME'),
-					pass: config('EMAIL_PASSWORD')
-				}
-			})
-		}
-
-		// mail service we decide to use in production
-		return nodemailer.createTransport({
-			service: config('MAIL_SERVICE'),
-			auth: {
-				user: config('MAIL_SERVICE_USERNAME'),
-				pass: config('MAIL_SERVICE_PASSWORD')
-			}
-		})
+const devConfig = {
+	host: config('EMAIL_HOST'),
+	port: config('EMAIL_PORT'),
+	auth: {
+		user: config('EMAIL_USERNAME'),
+		pass: config('EMAIL_PASSWORD')
 	}
+}
 
-	/**
-     * This method sends the mail
-     * @param {string} subject - the subject of the email
-     * @param {string} text - The body of the the email
-     */
-	async send(subject, text) {
-
-		const mailOptions = {
-			from: this.from,
-			to: this.to,
-			subject,
-			text 
-		}
-
-		await this.newTransport().sendMail(mailOptions)
+const prodConfig = {
+	service: config('MAIL_SERVICE'),
+	auth: {
+		user: config('MAIL_SERVICE_USERNAME'),
+		pass: config('MAIL_SERVICE_PASSWORD')
 	}
 }
+
+
+/** 
+ * Sends a mail 
+ * @param {string} from - sender
+ * @param {string} to - receiver
+ * @param {string} subject - subject of the mail
+ * @param {string} text - body of the mail
+ * @returns {Promise}
+*/
+const sendEmail = (from, to, subject, text) => {
```

Comment:
considering the number of arguments, lets use of object destructure to sort of get named parameters so that the order won't matter. Similar to how the `transporter.sendMail` is doing it

```suggestion
const sendEmail = ({ from, to, subject, text }) => {
```

also kindly provide a default for the `from` as well since there's one in the config